code/datums/mutantraces.dm

```diff
@@ -1126,7 +1127,10 @@ TYPEINFO(/datum/mutantrace)
 		. = ..()
 
 	proc/set_head(var/head)
-		src.head = head
+		if (src.head_tracker != null)
+			src.head_tracker.linked_human = null
```

Comment:
```suggestion

```
Just realized that this isn't necessary since you're setting it to something else right after anyways.